runtime/bcutil/attrlookup.h

```diff
@@ -1,27 +1,5 @@
-/*******************************************************************************
```

Comment:
Is there a reason to remove the copyright ?

Reply:
This is autogenerated from the changes attrlookup.gperf

Reply:
Does it mean that the existing copyright was incorrect added ? 

@pshipton @keithc-ca Do you know what is OpenJ9's rule of putting copyright on the generated code ?

Reply:
I think we should keep the copyright notice, because there are changes we want to keep relative to the generated content.